### PR TITLE
docs: add assert migration guide

### PR DESCRIPTION
**Pull request by stijnmoreels**
Add a detailed migration guide to move from the Testing Framework output comparison/XSLT to the Arcus assertion library.

Relates to #101 
Follow-up of #102

### CODE REVIEW

**Comment by stijnmoreels** [comment]
First version, will reiterate once the CSV assertion is also done.

File: docs/preview/03-Guidance/migrate-from-testing-framework.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+
+- using Stream expectedJsonStream = ...;
+- using Stream actualJsonStream = ...;
+- bool isEqual = Json.Compare(
+-    actualJsonStream, expectedJsonStream, out string userMessage, nodesToIgnore: Array.Empty<string>());
+
++ AssertJson.Equal(expectedJson, actualJson);
+```
+
+Any nodes that should be ignored can be configured with passing additional options:
```
Comment (ClementVaillantCodit) [inline docs/preview/03-Guidance/migrate-from-testing-framework.md:65]: "can be configured with passing"
I would say "_by_ passing"

**Review by ClementVaillantCodit — APPROVED** [review]
Awesome!!

File: docs/preview/03-Guidance/migrate-from-testing-framework.md (hunk 1 above)
Comment (stijnmoreels) [reply docs/preview/03-Guidance/migrate-from-testing-framework.md]: Yeah, is better, thx! Fixed that here: https://github.com/arcus-azure/arcus.testing/pull/109/commits/fe6ab4ad368aacde4b6935f5ebf5f86ad54982cb

**Pull request merged**